src/components/Header.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Header = () => (
+  <header>
+    <h1>Bookstore</h1>
+    <nav>
+      <Link to="/" className="nav-link">BOOKS</Link>
+      <Link to="/categories" className="nav-link">CATEGORIES</Link>
+    </nav>
+  </header>
```

Comment:
:arrow_forward: Kindly make sure that your Design is EXACTLY the same as the one in Zeplin. Color, font size, font weight, layouts, and orientations are exactly the same.
- [ ] Kindly distinguish between the header and the book container as seen in the Zeplin design. The color and font-weight of the `Books` and `CATEGORIES`  links should be EXACTLY the same as in the Zeplin design. Make sure that the user icon is present in your design. Also, align the `h1`(logo) so it should be in line with the book container. The annotations on the screenshot below may be helpful to you.

Your Design | Zeplin
:---:|:---: 
![](https://user-images.githubusercontent.com/63087491/178725133-1c6870dd-780b-4922-ad26-83384e530014.png) | ![](https://user-images.githubusercontent.com/63087491/178724712-8e8a81a5-3725-4883-a75d-80e45b7d80a6.png) 

- [ ] The button ADD BOOK should be align vertically as seen in the Zeplin design.